src/internal/operators/ignoreElements.ts

```diff
@@ -13,7 +14,7 @@ import { MonoTypeOperatorFunction } from '../types';
  * @method ignoreElements
  * @owner Observable
  */
-export function ignoreElements<T>(): MonoTypeOperatorFunction<T> {
+export function ignoreElements<T>(): OperatorFunction<T, Observable<never> > {
```

Comment:
An `OperatorFunction<A, B>` is a function that takes an `Observable<A>` and returns an `Observable<B>` like so: `(source: Observable<A>) => Observable<B>`. What you've defined here is `(source: Observable<T>) => Observable<Observable<never>>`, which isn't quite what we want.

It doesn't really matter what type of values are coming out of the source observable, so that can be `Observable<any>`, and we're returning an `Observable<never>`, so the function we want is: `(source: Observable<any>) => Observable<never>`. Given that, we want to return `OperatorFunction<any, never>` 😄 

This should really be:

```ts
export function ignoreElements(): OperatorFunction<any, never> {
}
```

Reply:
I didn't notice it was resolving to `Observable<Observable<never>>` :flushed: 

I'm not sure how to test for proper types yet